example/main.tf

```diff
@@ -2,16 +2,42 @@
 ## defaults
 ################################################################################
 terraform {
-  required_version = "~> 1.3, < 2.0.0"
+  required_version = ">= 1.0"
```

Comment:
Can we move this up to the latest version since we won't be supporting anything lower than 1.5?

Reply:
I tried, but it seems the restriction is coming from the underlying cloudposse module
![image](https://github.com/sourcefuse/terraform-aws-arc-security/assets/43397610/36624700-1c32-4f5b-beb5-ba7726299638)

https://github.com/cloudposse/terraform-aws-security-hub

We could fork and update